tools/build_tar/buildtar_test.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
not sure how I missed it before, but I think this file needs a boilerplate header.